src/pages/reports/index/Reports.tsx

```diff
@@ -470,6 +499,18 @@ export function Reports() {
               />
             </Element>
           )}
+
+          {report.identifier === 'product_sales' && (
+            <Element leftSide={t('client')}>
+              <ClientSelector
+                value={report.payload.client_id}
+                onChange={(client) =>
+                  handlePayloadChange('client_id', client.id)
+                }
+                withoutAction={true}
+              />
+            </Element>
```

Comment:
It looks like we are missing a clear button.

Reply:
Good catch, it's added!